File: c++/fibonacci/recursive_fib.cpp
#include <iostream>
#include <vector>

int fib(int pos) 
{
    if (pos < 2) 
    {
        return pos;
    }
    return fib(pos - 2) + fib(pos - 1);
}

int main()
{
    std::cout << fib(4);
}

File: c#/fibonacci/recursive_fib.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace json_app
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine(recursiveFib(4));
            Console.ReadKey();
        }
        static int recursiveFib(int pos)
        {
            if (pos < 2)
            {
                return pos;
            }
            return recursiveFib(pos - 2) + recursiveFib(pos - 1);
        }
    }
}
